sdk/synapse/azure-synapse/azure/synapse/aio/_synapse_client_async.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import Any
+
+from azure.core import AsyncPipelineClient
+from msrest import Deserializer, Serializer
+
+from ._configuration_async import SynapseClientConfiguration
+from .operations_async import MonitoringOperations
+from .operations_async import SparkBatchOperations
+from .operations_async import SparkSessionOperations
+from .operations_async import WorkspaceAclOperations
+from .. import models
+
+
+class SynapseClient(object):
+    """
+
+    :ivar monitoring: MonitoringOperations operations
+    :vartype monitoring: azure.synapse.aio.operations_async.MonitoringOperations
+    :ivar spark_batch: SparkBatchOperations operations
+    :vartype spark_batch: azure.synapse.aio.operations_async.SparkBatchOperations
+    :ivar spark_session: SparkSessionOperations operations
+    :vartype spark_session: azure.synapse.aio.operations_async.SparkSessionOperations
+    :ivar workspace_acl: WorkspaceAclOperations operations
+    :vartype workspace_acl: azure.synapse.aio.operations_async.WorkspaceAclOperations
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: azure.core.credentials.TokenCredential
+    :param synapse_dns_suffix: Gets the DNS suffix used as the base for all Synapse service requests.
+    :type synapse_dns_suffix: str
+    :param livy_api_version: Valid api-version for the request.
+    :type livy_api_version: str
+    """
+
+    def __init__(
+        self,
+        credential: "TokenCredential",
+        synapse_dns_suffix: str,
```

Comment:
Can we set default value "dev.azuresynapse.net" for synapse_dns_suffix? Just like livy_api_version

Reply:
fixed